src/desktop/src/ui/views/onboarding/SeedIntro.js

```diff
@@ -74,19 +79,10 @@ class SeedIntro extends React.PureComponent {
                             </p>
                         </Trans>
                     </Info>
-                    <p className={classNames(css.usb, ledger ? css.on : null)}>
```

Comment:
On smaller screens the heading might need to be moved down:

![image](https://user-images.githubusercontent.com/19519564/56856955-624dea00-6934-11e9-9ab8-68dd083371cc.png)

It looks fine in full screen however:
![image](https://user-images.githubusercontent.com/19519564/56856967-8e696b00-6934-11e9-9b03-4ebbbf85e854.png)